sql/src/test/java/io/crate/integrationtests/JoinGroupByIntegrationTests.java

```diff
@@ -137,20 +138,15 @@ public void testDistributedJoinWithAggregationScalarFunctionWithGroupBy() throws
 
     @Test
     public void testHavingWithGroupBy() throws Exception {
+        expectedException.expect(SQLActionException.class);
+        expectedException.expectMessage("Column \"name\" is ambiguous");
```

Comment:
I think this test should be adapted so that it still works - otherwise we're losing a test-case for group by with having on joins. Changing the GROUP BY keys to be fully qualified should work?